terms/abs/abs.md

```diff
@@ -0,0 +1,51 @@
+## Syntax
```

Comment:
This entry is missing a metadata section. Please review the [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md#metadata) to see what is needed.

Reply:
I will update it. Thanks for the heads up.